orbit/CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## Orbit 1.28.0 (Jul 18, 2024)
+
+* Fixes a bug that caused log Orbit's osquery table log output to be inconsistent.
+
+adding support for new agent option `script_execution_timeout` to configure seconds until a script
+is killed due to timeout.
+
+* Updated Go version to go1.22.4
+
+* Fix boot loop caused by Linux hosts with no hardware UUID
+
+* Added support for Linux ARM64
+
+* Fix bug where UTC timezone could cause error in `fleetd_logs` table time parsing
```

Comment:
pending confirmation, we think we're missing the changes to hide the 'Self-service' menu item if you don't have software installers